content/account/users.textile

```diff
@@ -74,4 +74,27 @@ To remove a user from your account:
 
 h2(#delete). Delete your account
 
-Follow "this process":https://faqs.ably.com/how-do-i-cancel-my-account to delete your Ably account. Only an account owner can initiate this process.
+You can cancel your account at any time by downgrading to the "Free package":/docs/pricing/free. The following steps outline the the process to cancel your account:
+
+* Ensure you are the account "owner":/docs/account/users#roles.
+* Downgrade all paid "packages":/docs/pricing#packages free packages:
```

Comment:
This is referring to multiple packages throughout the steps, but presumably this happens on an account-by-account basis and should be singular?